source/carla_host.py

```diff
@@ -1008,6 +1008,11 @@ def slot_pluginRemoveAll(self):
         if self.fPluginCount == 0:
             return
 
+        ask = QMessageBox.question(self, self.tr("Remove All"), self.tr("Are you sure you want to remove all plugins?"), QMessageBox.Yes|QMessageBox.No)
```

Comment:
multiple lines here please, and no need to save the return value, just compare directly to the No option.

Reply:
Maybe it's just me, but I think this looks a bit less clear without saving the return value:
```
if QMessageBox.question(self, self.tr("Remove All"), 
                              self.tr("Are you sure you want to remove all plugins?"), 
                              QMessageBox.Yes|QMessageBox.No) == QMessageBox.No:
                                            return
```

Reply:
I still prefer this version tbh